lizrd/train/train_utils.py

```diff
@@ -88,12 +86,10 @@ def get_model(
 
     if activation_checkpointing_modules is not None:
         check_fn = lambda x: isinstance(x, activation_checkpointing_modules)
-        non_reentrant_wrapper = partial(
-            checkpoint_wrapper,
-            checkpoint_impl=CheckpointImpl.NO_REENTRANT,
```

Comment:
what happened to NO_REENTRANT

Reply:
ah, I see that it's got `use_reentrant = False`. What is the difference?

Reply:
That's the same thing.